packages/@aws-cdk/aws-rds/lib/instance.ts

```diff
@@ -384,22 +384,22 @@ export interface DatabaseInstanceNewProps {
   /**
    * The amount of time, in days, to retain Performance Insights data.
    *
-   * @default 7 days
+   * @default 7
```

Comment:
same

Reply:
This one is an enum (`PerformanceInsightRetention`), I think it's correct?